ephemeralkey.go

```diff
@@ -19,10 +19,6 @@ type EphemeralKeyParams struct {
 }
 type EphemeralKey struct {
 	APIResource
-	AssociatedObjects []struct {
```

Comment:
The changelog entry just says "Removed deprecated EphemeralKey field." Should we talk more about the AssociatedObjects?

Reply:
Didn't finish typing..